README.md

```diff
@@ -10,10 +10,10 @@ We are running an IPFS-Cluster with the following endpoints publicly available:
 
   - ![POST](https://img.shields.io/badge/-POST-green "POST") `https://api.ipfs.lukso.network`[/api/v0/add](https://docs.ipfs.io/reference/http/api/#api-v0-add)
   - ![POST](https://img.shields.io/badge/-POST-green "POST") `https://api.ipfs.lukso.network`[/api/v0/block](https://docs.ipfs.io/reference/http/api/#api-v0-add)
```

Comment:
Wasn't sure which link to use here as there is no `/api/v0/block` but `/api/v0/block/[get,put,rm,stat]`

Reply:
https://docs.ipfs.io/reference/http/api/#api-v0-block-get